### ISSUE TITLE
Battery stat doesn't work

### ISSUE THREAD

**Issue by labliann** [wasdennnoch/AndroidN-ify#997]
We need to click edit tile to work the battery stat.

Video:
https://www.dropbox.com/s/avf3qtiq2jtram6/SCR_20160928_225525.mp4?dl=0

Xposed log:
https://www.dropbox.com/s/wmjvs2p2dc6bxu9/xposed_error_20160928_225548.log?dl=0

**Comment by labliann** [comment]
Chroma AOSP ROM

**Comment by labliann** [comment]
Google messenger quick reply
https://www.dropbox.com/s/1gab4shrtbjofyu/Screenshot_20160929-072533.png?dl=0

https://www.dropbox.com/s/1vg442i654bab84/Screenshot_20160929-072615.png?dl=0

**Comment by wasdennnoch** [comment]
Does the battery bug still happen?

**Comment by labliann** [comment]
Yes still happen same in 732 version and if i add tile doesn't save

**Comment by wasdennnoch** [comment]
Please post a new log then as the old one is inaccessible.

**Comment by labliann** [comment]
androidN.ify 747

Video:
https://www.dropbox.com/s/ynt8is5eljk3k4u/SCR_20161106_082214.mp4?dl=0

Xposed log:
https://www.dropbox.com/s/kz7vpt4hz0jxdo5/xposed_error_20161106_082023.log?dl=0

I think this video can explain clearly
